kitsune/users/tests/test_templates.py

```diff
@@ -157,6 +157,13 @@ def test_ga_custom_variable_on_admin_login(self):
         doc = pq(response.content)
         assert '"Contributor - Admin"' in doc('body').attr('data-ga-push')
 
+    def test_login_mobile_csrf(self):
+        """The mobile login view should have a CSRF token."""
+        response = self.client.get(reverse('users.login'), {'mobile': 1})
+        eq_(200, response.status_code)
+        doc = pq(response.content)
+        assert doc('input[name="csrfmiddlewaretoken"]')
```

Comment:
Some of our pages have csrf tokens stashed elsewhere for JS. It would probably be better to target this to a particular form.